src/content/app/tools/blast/state/blast-api/blastApiSlice.ts

```diff
@@ -66,9 +67,14 @@ export type RejectedJob = {
 const blastApiSlice = restApiSlice.injectEndpoints({
   endpoints: (builder) => ({
     blastConfig: builder.query<BlastSettingsConfig, void>({
-      query: () => ({
-        url: `${config.toolsApiBaseUrl}/blast/config`
-      }),
+      queryFn: () => {
+        return Promise.resolve({
+          data: mockConfig as unknown as BlastSettingsConfig
+        });
+      },
+      // query: () => ({
+      //   url: `${config.toolsApiBaseUrl}/blast/config`
+      // }),
```

Comment:
**REMINDER TO US ALL:** remember to change this back after the backend has updated the config